.github/workflows/release.yml

```diff
@@ -8,9 +8,12 @@ on:
     tags:
       - "v*.*.*"
 
+permissions:
+  contents: write
```

Comment:
Since this is creating a release and uploading assets, we still need `contents: write`.